release/models/sampling/openconfig-sampling-sflow.yang

```diff
@@ -30,13 +30,10 @@ module openconfig-sampling-sflow {
     RFC 3176 - InMon Corporation's sFlow: A Method for
     Monitoring Traffic in Switched and Routed Networks";
 
-  revision "2022-06-21" {
+  revision "2025-05-13" {
```

Comment:
You'll need to leave the old revision intact and merely add the new revision.  So both 1.0.0 and 1.1.0 should appear here